Index: backend-base/src/main/webapp/js/lib/scriptaculous/controls.js
===================================================================
--- backend-base/src/main/webapp/js/lib/scriptaculous/controls.js	(revisión: 18625)
+++ backend-base/src/main/webapp/js/lib/scriptaculous/controls.js	(copia de trabajo)
@@ -60,7 +60,7 @@
     this.options.paramName    = this.options.paramName || this.element.name;
     this.options.tokens       = this.options.tokens || [];
     this.options.frequency    = this.options.frequency || 0.4;
-    this.options.minChars     = this.options.minChars || 1;
+    this.options.minChars     = Object.isNumber(this.options.minChars) ? this.options.minChars : 1;
     this.options.onShow       = this.options.onShow ||
       function(element, update){
         if(!update.style.position || update.style.position=='absolute') {
